File: 3-/ex7.py
print("==Calculo de Media==")
print()

quant = int(input("Quantas notas você vai informar? "))
soma = 0

for i in range(1, quant + 1):
    nota = float(input(f"Digite a nota {i}: "))
    soma += nota

media = soma / quant
print("Média:", media)

if media >= 7:
    print("Situação: Aprovado")
elif media >= 5:
    print("Situação: Recuperação")
else:
    print("Situação: Reprovado")

File: 3-Repeticao/ex7.php
<?php
echo "Quantas notas você vai informar? ";
$quant = (int)trim(fgets(STDIN));
$soma = 0;

for ($i = 1; $i <= $quant; $i++) {
    echo "Digite a nota $i: ";
    $nota = (float)trim(fgets(STDIN));
    $soma += $nota;
}

$media = $soma / $quant;
echo "Média: $media\n";

if ($media >= 7) {
    echo "Situação: Aprovado\n";
} elseif ($media >= 5) {
    echo "Situação: Recuperação\n";
} else {
    echo "Situação: Reprovado\n";
}
?>
